packages/create-llama/create-app.ts

```diff
@@ -100,7 +101,13 @@ export async function createApp({
     });
     // copy readme for fullstack
     await fs.promises.copyFile(
-      path.join(__dirname, "..", "templates", "README-fullstack.md"),
+      path.join(
+        fileURLToPath(import.meta.url),
+        "..",
+        "..",
+        "templates",
+        "README-fullstack.md",
+      ),
```

Comment:
Better to calculcate `templatesDir` once and use it everywhere
```suggestion
      path.join(
        templatesDir,
        "README-fullstack.md",
      ),
```